Description: install *all* the HTML docs
 The relevant makefile target was never updated since 2004..
 Should probably look for a nicer way to do this than the current list before forwarding.
Author: Mattia Rizzolo <mattia@debian.org>
Last-Update: 2021-07-28
Forwarded: no

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -334,9 +334,55 @@
 
 rebuild: api docs
 
+HTML_TOP_DIR= \
+	architecture.html \
+	bugs.html \
+	catalog.gif \
+	catalog.html \
+	contribs.html \
+	docs.html \
+	downloads.html \
+	DOM.gif \
+	DOM.html \
+	encoding.html \
+	entities.html \
+	example.html \
+	FAQ.html \
+	index.html \
+	interface.html \
+	intro.html \
+	guidelines.html \
+	help.html \
+	library.html \
+	libxml.gif \
+	libxml2-api.xml \
+	Libxml2-Logo-90x34.gif \
+	Libxml2-Logo-180x168.gif \
+	namespaces.html \
+	news.html \
+	python.html \
+	redhat.gif \
+	searches.html \
+	smallfootonly.gif \
+	structure.gif \
+	threads.html \
+	tree.html \
+	upgrade.html \
+	w3c.png \
+	xml.html \
+	xmlcatalog_man.html \
+	xmldtd.html \
+	xmlio.html \
+	xmllint.html \
+	xmlmem.html \
+	xmlreader.html \
+	XMLinfo.html \
+	XSLT.html
+
+
 install-data-local: 
 	$(MKDIR_P) $(DESTDIR)$(HTML_DIR)
-	-$(INSTALL) -m 0644 $(srcdir)/xml.html $(srcdir)/encoding.html $(srcdir)/FAQ.html $(srcdir)/structure.gif $(srcdir)/DOM.gif $(srcdir)/smallfootonly.gif $(srcdir)/redhat.gif $(srcdir)/libxml.gif $(srcdir)/w3c.png $(srcdir)/Libxml2-Logo-180x168.gif $(srcdir)/Libxml2-Logo-90x34.gif $(DESTDIR)$(HTML_DIR)
+	-$(INSTALL) -m 0644 $(foreach file,$(HTML_TOP_DIR),$(srcdir)/$(file)) $(DESTDIR)$(HTML_DIR)
 	$(MKDIR_P) $(DESTDIR)$(HTML_DIR)/html
 	-$(INSTALL) -m 0644 $(srcdir)/html/*.html $(DESTDIR)$(HTML_DIR)/html
 	-$(INSTALL) -m 0644 $(srcdir)/html/*.png $(DESTDIR)$(HTML_DIR)/html
